dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task230523CreateVariantFieldInContentletIntegrationTest.java

```diff
@@ -0,0 +1,120 @@
+package com.dotmarketing.startup.runonce;
+
+
+import static junit.framework.TestCase.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+import com.dotcms.contenttype.model.type.ContentType;
+import com.dotcms.datagen.ContentTypeDataGen;
+import com.dotcms.datagen.ContentletDataGen;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotcms.util.JsonUtil;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.common.db.DotDatabaseMetaData;
+import com.dotmarketing.db.DbConnectionFactory;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+import com.dotmarketing.util.UtilMethods;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Map;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class Task230523CreateVariantFieldInContentletIntegrationTest {
+
+
+    @BeforeClass
+    public static  void variantIdExists() throws Exception {
+
+        // Setting web app environment
+        IntegrationTestInitService.getInstance().init();
+
+        final Connection connection = DbConnectionFactory.getConnection();
+        final ResultSet resultSet = DotDatabaseMetaData.getColumnsMetaData(connection, "contentlet");
+        boolean existsVariantId = false;
+
+        while(resultSet.next()){
+            final String columnName = resultSet.getString("COLUMN_NAME");
+            final String columnType = resultSet.getString(6);
+
+            if (columnName.equals("variant_id")) {
+                assertEquals(columnType, "varchar");
+                existsVariantId = true;
+            }
+        }
+
+        assertTrue(existsVariantId);
+    }
+
+    @Test
```

Comment:
I would add a javadoc or some comments on the test to help understand was it's intended for